app/templates/public/index.hbs

```diff
@@ -44,14 +44,6 @@
     </div>
     <div class="ui hidden divider"></div>
   {{/if}}
-  {{#if nonFeaturedSpeakers}}
```

Comment:
@agbilotia1998 this change is being done in https://github.com/fossasia/open-event-frontend/pull/1500.